### ISSUE TITLE
Content pages cannot be accessed with URL

### ISSUE THREAD

**Issue by Mackiovello** [StarcounterApps/Content#30]
### Steps to reproduce

Build and run. Go to `http://localhost:8080/content/dynamic/index/header` or any other page. See that there is no content there.

### Expected result

The [Header page](https://github.com/StarcounterApps/Content/blob/master/src/Content/wwwroot/Content/dynamic/index/HeaderPage.html) should be displayed

### Actual result

Nothing is displayed.

The response looks like this:
```html 
<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
    <title>Content</title>

    <script src="/sys/webcomponentsjs/webcomponents.min.js"></script>
    <script>
      /* this script must run before Polymer is imported */
      /*
       * Let Polymer use native Shadow DOM if available.
       * Otherwise (at least Polymer 1.x) assumes everybody else
       * uses ShadyDOM, which is not true, as many Vanilla CE uses
       * real Shadow DOM.
       */
      window.Polymer = {
        dom: "shadow"
      };
    </script>
    <link rel="import" href="/sys/polymer/polymer.html">
    <link rel="import" href="/sys/starcounter.html">
    <link rel="import" href="/sys/starcounter-include/starcounter-include.html">
    <link rel="import" href="/sys/starcounter-debug-aid/src/starcounter-debug-aid.html">

    <link rel="import" href="/sys/bootstrap.html">
    <style>
        body {
            margin: 20px;
        }
    </style>
</head>
<body>
    <template is="dom-bind" id="puppet-root">
        <template is="imported-template" content$="{{model.Html}}" model="{{model}}"></template>
    </template>

    <puppet-client ref="puppet-root" remote-url="/__default/BEB7CA9C29F3CF8D20000010"></puppet-client>

    <starcounter-debug-aid></starcounter-debug-aid>
</body>
</html>
```
The view-model looks like this: 

```json
{
  "_ver#c$": 0,
  "_ver#s": 0,
  "Pages": [
    {
      "Content": {
        "Html": "/content/dynamic/index/HeaderPage.html",
        "User": {
          "Username": "admin"
        }
      }
    }
  ]
}
```

### Technical information

Starcounter version 2.3.0.6256

It seems like the handler in [ContentHandlers](https://github.com/StarcounterApps/Content/blob/master/src/Content/Api/ContentHandlers.cs) is called twice.

**Comment by Mackiovello** [comment]
This is my bad. The right URL to call is `/content/cms/item/{key}`

Closing

**Comment by Mackiovello** [comment]
The above link only takes me to the edit page, not the actual page.

Reopening

**Comment by Mackiovello** [comment]
Fixed with 	a2106f6

Closing
